src/libmongoc/tests/unified/result.c

```diff
@@ -564,7 +558,7 @@ test_resultfrombulkwrite (void)
       "'modifiedCount': 3, 'upsertedCount': 4}");
    result_from_bulk_write (result, reply, &empty);
    MONGOC_DEBUG ("rewritten to: %s", bson_val_to_json (result->value));
-   if (!result_check (result, false, NULL, expect, NULL, &error)) {
+   if (!result_check (result, NULL, expect, NULL, &error)) {
```

Comment:
I realize `false` was incorrect since the second param is a pointer to the entity map. Is `NULL` acceptable here because this isn't part of the general unified test runner (where an operator might need the entity map) and instead just an isolated unit test?

Reply:
This particular diff was from https://github.com/mongodb/mongo-c-driver/pull/820/commits/7679dc7085cbd178f00062f07fe9195724244db3 where I simplified the implementation. The `NULL` was there before, and I assumed the same that you do. @kevinAlbs can you confirm it's intentional that we're not passing the entity map?

Reply:
Oh, in that case this may be like what we do in [tests/UnifiedSpecTests/Constraint/MatchesTest.php](https://github.com/mongodb/mongo-php-library/blob/master/tests/UnifiedSpecTests/Constraint/MatchesTest.php). The constraint tests that don't actually need to resolve anything in the EntityMap will pass in null.

Reply:
It is intentional. This is a self test of the `result_check` method. @jmikola is correct. The test assertion is not referencing anything in the entity map (e.g. `$$matchesEntity`)